.gitignore

```diff
@@ -1,2 +1,2 @@
 node_modules
-.env
+.env
```

Comment:
Would be cool to reset this file so it is not included in the PR. Not a big deal if that ends up being challenging.

Reply:
not sure how to do it? i will try more tho